tests-e2e/step-definitions/standard-ui/form.js

```diff
@@ -143,7 +143,7 @@ When('I click on vue checkbox control with id {string}',
   async (id) => {
     client.useXpath();
 
-    const selector = `//*[@id="${id}"]/following-sibling::label[@for="${id}"]`;
```

Comment:
The label always having a `for` attribute would be a good thing. Would be doing it another way be too much trouble?

Reply:
Described in the first comment.